storage/stonedb/index/kv_store.cpp

```diff
@@ -46,7 +46,7 @@ void KVStore::Init() {
   // get column family names from manifest file
   rocksdb::Status status = rocksdb::DB::ListColumnFamilies(db_option, rocksdb_datadir, &cf_names);
   if (!status.ok() &&
-      ((status.subcode() == rocksdb::Status::kNone) || (status.subcode() == rocksdb::Status::kPathNotFound))) {
```

Comment:
Do not REMOVE the condition `status.subcode() == rocksdb::Status::kPathNotFound`